src/txkube/testing/_eliot.py

```diff
@@ -5,7 +5,7 @@
 
 from fixtures import Fixture
 
-from eliot import add_destination
+from eliot import add_destination, remove_destination
```

Comment:
Yep.  This is only half the fix, as it turns out.  I noticed the problem at some point during development of #5 - there is actually no `Fixture._tearDown`  :(  Instead, the code should `self.addCleanup(...)` inside `_setUp`.  Feel like doing the rest of that here?